tests/contrib/futures/test_propagation.py

```diff
@@ -1,14 +1,24 @@
-import concurrent
+import concurrent.futures
```

Comment:
Do we need to update this import?

Reply:
my editor wasn't happy, and this is technically more correct, since `futures` might not be imported by `concurrent`.

"no", but 🤷🏻